nixos/modules/services/web-apps/nextcloud.nix

```
@@ -514,6 +514,30 @@ in {
               `http://hostname.domain/bucket` instead.
             '';
           };
+          useSseC = mkOption {
+            type = types.bool;
+            default = false;
+            description = lib.mdDoc ''
+              Use [server-side encryption with customer-provided keys][1].
+
+              A random 32-byte key must be provided as a base64 string, e.g.
+              with the command
+
+              ```
+              openssl rand 32 | base64
+              ```
+
+              [1]: https://docs.aws.amazon.com/AmazonS3/latest/userguide/ServerSideEncryptionCustomerKeys.html
```

```suggestion
              Use [server-side encryption with customer-provided keys][1].

              A random 32-byte key must be provided as a base64 string.
              One can be generated with the following command:
              ```
              openssl rand 32 | base64
              ```

              [1]: https://docs.aws.amazon.com/AmazonS3/latest/userguide/ServerSideEncryptionCustomerKeys.html
```

---

Thank you. Please check the new description after the removal of `useSseC` [here](https://github.com/NixOS/nixpkgs/blob/8008e73c3878ec4b4e9633c89c5113c1cfc6abbe/nixos/modules/services/web-apps/nextcloud.nix#L521).